charts/immich/templates/checks.yaml

```diff
@@ -1,7 +1 @@
 {{- $name := .Values.immich.persistence.library.existingClaim | required ".Values.immich.persistence.library.existingClaim is required." -}}
-
-{{ if .Values.postgresql.enabled }}
-  {{ if not .Values.useDeprecatedPostgresChart}}
-    {{ fail "The postgres subchart is deprecated. Please see https://github.com/immich-app/immich-charts/issues/149 for more detail." }}
-  {{ end }}
-{{ end }}
```

Comment:
I'm wondering whether we should convert this to fail on `if .Values.postgresql` at all to catch any stragglers?

Reply:
Yup, good shout. Implemented [here](https://github.com/immich-app/immich-charts/pull/190/commits/bbe803b8540ccf918536d6af3e393f52d69412cc)